src/app/project/add.tpl.html

```diff
@@ -39,6 +39,7 @@
                        ng-model="vm.project_name"
                        ng-model-options="{ debounce: 500 }"
                        project-name-available-validator
+                       organization-id="vm.currentOrganization.id"
```

Comment:
It will result in `__newOrganization`, which seems to be set in the [project controller](https://github.com/exceptionless/Exceptionless.UI/blob/a436aac/src/app/project/add-controller.js#L121).

The actual result is mentioned in my comment https://github.com/exceptionless/Exceptionless.UI/pull/123#issuecomment-276879758